JetStreamDriver.js

```diff
@@ -2357,13 +2352,13 @@ function enableBenchmarksByTag(tag, forceEnable = false)
     }
 
     for (const benchmark of benchmarks) {
-        if (!forceEnable && benchmark.disabledByDefault)
-            continue;
-
+        if (excludeTags && benchmark.hasAnyTag(...excludeTags))
+            continue
         JetStream.addBenchmark(benchmark);
```

Comment:
I think this doesn't deduplicate tests. We should probably push the benchmarks into a `Set` then `Array.from` them at the end. e.g. running

```% jsc cli.js -- default Wasm
Starting JetStream3
Running zlib-wasm:
    Startup: 108.743
    Worst Case: 115.567
    Average: 116.799
    Score: 113.647
    Wall time: 0:01.718
zlib-wasm
Running zlib-wasm:
JetStream3 failed: Error: Cannot run Benchmark zlib-wasm twice
```

This seems like something folks might end up doing relatively easily if/when we add more tags.

Reply:
Ah, sorry failed to copy that line:
`% jsc cli.js -- default Wasm`